apps/wrf/README.md

```diff
@@ -104,7 +104,7 @@ You can accomplish this by typing:
 
```

Comment:
Update Intel oneAPI compiler and Intel oneAPI MPI 2021.3.0
- GNU 10.2.0 and Open MPI 4.1.0  :  to the latest versions

Reply:
it will be address in a later pull request, see #25 , #26, #27